### DISCUSSION TITLE
Alternative BLDC motors?

### DISCUSSION THREAD

**Discussion by adamtime479** [upkie/upkie#464] [Hardware] [answered]
Hello!

I was wondering if anyone made the part list alot cheaper, since the mjbots motors and parts in general are quite expensive.
If so, anyone wanna share with me here or privately what they did?
Otherwise i might have to F*** around and find out but Im broke yo, so Im scared.

I was thinking of hitting amazon on black friday to get some cheap BLDC motors anyways and hopefully finding a cheap pair with integrated encoders so I can do some reinforced learning smoothely.

Well thanks in advance.

**Comment by stephane-caron** [comment, marked as answer]
You could try to reproduce a [MABEL](https://github.com/raspibotics/MABEL) wheeled biped?

The list of [Awesome Open Source Robots](https://github.com/stephane-caron/awesome-open-source-robots) sums up all the other open source robots I have found so far, ranging from cheaper ones with stepper motors to more expensive research ones with e.g. high-end 3D printing involved. Feel free to check it out, some of the projects there may be a good fit to what you are looking for.
